stores/mongodb/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './vector';
 export * from './storage';
+export * from './storage/ConnectorHandler';
```

Comment:
this doesnt exist? we moved to diff files

Reply:
Fixed @abhiaiyer91